app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Rooms Index</h1>
+
+<ul>
+  <% @rooms.each do |room| %>
+    <li>
+      <p><strong>Hotel: <%= room.hotel.name %>,</strong></p>
+      <p>Suite: <%= room.suite %>,</p>
+      <p>Rate: $<%= room.rate %>,</p>
+      <p>Number of Stays: <%= room.stays.count %></p>
```

Comment:
Even though this is a simple AR calculation, it's still a db query that queries the database directly. Per MVC, we want to encapsulate this at the model level so that we can 1) keep querying beyond simple CRUD at the level closest to the db and 2) unit test it in  model specs! 

You could create a `guest_count` method in the Room model (and test it in room_spec) and then call it here with `room.guest_count`

Reply:
This is a minor MVC infraction but the rubric allows for one infraction!